web_src/js/components/RepoActionView.vue

```diff
@@ -443,37 +435,32 @@ export function initRepositoryActionView() {
 
 .job-step-section {
   margin: 10px;
-  .job-step-logs {
-    font-family: monospace, monospace;
-    .job-log-line {
-      display: flex;
-      .line-num {
-        width: 48px;
-        color: var(--color-grey-light);
-        text-align: right;
-      }
-      .log-time {
-        color: var(--color-grey-light);
-        margin-left: 10px;
-        white-space: nowrap;
-      }
-      .log-msg {
-        flex: 1;
-        word-break: break-all;
-        white-space: break-spaces;
-        margin-left: 10px;
-      }
-    }
+}
 
-    // TODO: group support
-    .job-log-group {
-    }
+.job-step-section .job-step-logs {
+  font-family: monospace;
+}
 
-    .job-log-group-summary {
-    }
+.job-step-section .job-step-logs .job-log-line {
+  display: flex;
+}
 
-    .job-log-list {
-    }
-  }
```

Comment:
Ah, we simply threw out this TODO?

Reply:
Yes, the online less compiler used for `.vue` files threw out comments but I will re-run these through `lessc` and check that all comments and empty rules are preserved.

Reply:
TODOs are back in original style with empty selectors.